src/common/fileUtils.js

```diff
@@ -0,0 +1,21 @@
+// TODO: add doc blocks
+
+export const dateCompare = (version1, version2) => {
+  return version1.node.createdAt < version2.node.createdAt;
```

Comment:
Make sure this get updated from the change in #216

Reply:
> Make sure this get updated from the change in #216

is this a general comment of the PR or specifically for the `dateCompare` function

Reply:
Just making sure it makes it through the rebase